src/ConnectFour/Application/Game/Query/Model/GamesByPlayer/GamesByPlayer.php

```diff
@@ -9,10 +9,12 @@
 final class GamesByPlayer
 {
     /**
+     * @param array<string, int> $totals
      * @param Game[] $games
      */
     public function __construct(
-        public readonly int $count,
+        public readonly array $totals,
```

Comment:
Although having this here is the most efficient way to build the page (since everything can run in the same Redis pipeline), it doesn’t belong here.